README.rst

```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
                                       --instance-type ml.m4.xlarge \
                                       --tag 1.3.0-cpu-py3
 
+Amazon Elastic Inference with MXNet in SageMaker
+------------------------------------------------
+`Amazon Elastic Inference <https://aws.amazon.com/machine-learning/elastic-inference/>`__ allows you to to attach
+low-cost GPU-powered acceleration to Amazon EC2 and Amazon SageMaker instances to reduce the cost running deep
+learning inference by up to 75%. Currently, Amazon Elastic Inference supports TensorFlow, Apache MXNet, and ONNX
+models, with more frameworks coming soon.
+
+Support for using MXNet with Amazon Elastic Inference in SageMaker is supported in the public `SageMaker MXNet containers <https://github.com/aws/sagemaker-mxnet-container>`__.
+
+* For information on how to use the Python SDK to create an endpoint with Amazon Elastic Inference and MXNet in SageMaker, see `Deploying MXNet Models <https://github.com/aws/sagemaker-python-sdk/tree/master/src/sagemaker/mxnet#deploying-mxnet-models>`__.
+* For information on how Amazon Elastic Inference works, see `How EI Works <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-how-it-works>`__.
+* For more information in regards to using Amazon Elastic Inference in SageMaker, see `Amazon SageMaker Elastic Inference <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html>`__.
+* For notebook examples on how to use Amazon Elastic Inference with MXNet through the Python SDK in SageMaker, see `EI Sample Notebooks <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-intro-sample-nb>`__.
+
+SageMaker Elastic Inference MXNet container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Amazon Elastic Inference is designed to be used with AWS enhanced versions of TensorFlow serving or Apache MXNet. These enhanced
+versions of the frameworks are automatically built into containers when you use the Amazon SageMaker Python SDK, or you can
+download them as binary files and import them into your own Docker containers. The enhanced MXNet binary are available on Amazon S3, `here <https://s3.console.aws.amazon.com/s3/buckets/amazonei-apachemxnet/>`__.
```

Comment:
* s/binary/binaries (for the last one)
* I'd remove "here" and make the link for "available on Amazon S3"

Reply:
in that case, I'd change it to "available on Amazon S3 at <link>" or "available on Amazon S3 here: <link>"

Reply:
Done.